pkg/spdk/util.go

```diff
@@ -209,3 +189,25 @@ func ExtractBackingImageAndDiskUUID(lvolName string) (string, string, error) {
 
 	return backingImageName, diskUUID, nil
 }
+
+// CleanupLvolTree retrieves the lvol tree with BFS. Then try its best effort to do cleanup bottom up.
+func CleanupLvolTree(spdkClient *spdkclient.Client, rootLvolName string, bdevLvolMap map[string]*spdktypes.BdevInfo, log *safelog.SafeLogger) {
+	var queue []*spdktypes.BdevInfo
```

Comment:
This function is only used in Replica. Why are we moving it to `util.go`? Or will it be reused in other places later?

Reply:
I also prefer this way. WDYT? @shuo-wu?

Reply:
In the future, it may be used by other components like `orphan`. Hence I prefer to retain it in `util.go`